### PR TITLE
build the "our team " component

### PR DESCRIPTION
**Pull request by junaid-rizwan**

### CODE REVIEW

**Comment by EmmanuelAllanMJ** [comment]
Fix the built error
image

File: package-lock.json
Comment (EmmanuelAllanMJ) [inline package-lock.json:1]: Remove package-lock when pushing as we are using yarn.

**Review by EmmanuelAllanMJ — COMMENTED** [review]
Reduce the resolution of images

**Comment by junaid-rizwan** [comment]
some changes

**Comment by EmmanuelAllanMJ** [comment]
> some changes

do yarn build or npm run build and fix the build error

**Pull request merged**